docs/releases/3.2.txt

```diff
@@ -244,6 +244,9 @@ Migrations
 * Migrations now support serialization of pure and concrete path objects from
   :mod:`pathlib`, and :class:`os.PathLike` instances.
 
+* Autodetection of migration creations no longer fails when the model uses a
+  custom metaclass.
+
```

Comment:
Refer to the class via ```:class:`~django.db.migrations.operations.CreateModel` ```  I'd also suggest explaining the new argument. Something along the lines of:

The new ``metaclass`` argument to :class:`~django.db.migrations.operations.CreateModel ...

Reply:
How about this:

> The migration framework now supports models with custom metaclasses and supports changing them.

Reply:
@smithdc1 thanks for the review! I made the changes, they're ready for another look